seahorse-workflow-executor/deeplang/src/main/scala/ai/deepsense/deeplang/TypeUtils.scala

```diff
@@ -57,10 +58,10 @@ object TypeUtils {
   }
 
   def instanceOfType[T](typeTag: ru.TypeTag[T]): T = {
-    val constructorT = constructorForType(typeTag.tpe).getOrElse {
+    val clazz = typeTag.mirror.runtimeClass(typeTag.tpe)
+    Try(clazz.newInstance.asInstanceOf[T]).getOrElse {
```

Comment:
Perhaps we could rework classToType and typeToClass methods to use appropriate mirrors instead?